src/time.rs

```diff
@@ -1,170 +1,114 @@
-use core::ops::{Add, Div};
+pub use fugit::{
```

Comment:
The contents of this file is now more or less identical with [the one from stm32g0xx-hal](https://github.com/stm32-rs/stm32g0xx-hal/blob/main/src/time.rs)